packages/lbvs_consent/lbvs_consent.2.1.1/opam

```diff
@@ -0,0 +1,65 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/consent"
+bug-reports: "https://github.com/UnixJunkie/consent/issues"
+dev-repo: "git+https://github.com/UnixJunkie/consent.git"
+license: "GPL"
+build: [
+  [make "obmaccs"]
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  ["cp" "bin/lbvs_consent_ecfp4.py" "%{bin}%/lbvs_consent_ecfp4.py"]
+  ["cp" "src/lbvs_consent_ob_maccs" "%{bin}%/lbvs_consent_ob_maccs"]
+]
+remove: [
+  ["rm" "-f" "%{bin}%/lbvs_consent_ecfp4.py"
+             "%{bin}%/lbvs_consent_ob_maccs"]
+]
```

Comment:
This field is unnecessary since opam 2.0

Reply:
Ok, I removed it.
Sorry, you probably already told me once before.
Opam lint doesn't say anything about it though.

Reply:
In practice, the remove field is useful now only if you modify some other files around (e.g. shared configurations) during install and you need to revert such changes in the remove phase. This means we cannot really deprecate it or raise a warning. See: https://opam.ocaml.org/doc/Manual.html#opamfield-remove

Can you please make the change upstream as well?

Reply:
ok, corrected upstream too